content.css

```diff
@@ -65,7 +65,7 @@
 }
 
 /* Hide "Trends for you" box */
-.bt--notrends [aria-labelledby="accessible-list-0"] {
+.bt--notrends [aria-labelledby^="accessible-list-"] > [aria-label="Cronología: Tendencias del momento"] {
```

Comment:
Thanks for trying to address this, but this approach won't work for the vast majority of users. 
The selector is targeting a language-dependent label. It won't work for other languages. 

This is already an issue with Better Twitter. It only targets labels in English. See #3 

What's needed is a language-agnostic approach, and generic enough not to be easily tripped by mundane changes in the markup. I don't yet have such an apporach, but I'm interested in ideas.

Reply:
I'll think on it ;)

Reply:
Nowaday we can asume that [aria-labelledby^="accessible-list-"] returns always two elements and the second one it's trends section, but I don't know how do this (properly). I tried with CSS but I don't know how to do this (I tried ~, nth-child...) and with JS I do the following and works fine except that when this function runs the first time DOM is not loaded and crashes...

`function applyPrefs(prefs = {}) {
  userPrefs = { ...DEFAULT_PREFS, ...prefs };
  Object.entries(userPrefs).forEach(([id, pref]) => {
    if (id === 'bt--notrends') {
      document.documentElement.classList.toggle(id, pref.value);
      const noTrendsSection = document.querySelectorAll('[aria-labelledby^="accessible-list-"]')[1];
      noTrendsSection.classList.toggle('bt--notrends--section', pref.value);
    } else {
      document.documentElement.classList.toggle(id, pref.value);
    }
  })
}`